coordinator/src/services/translation/request/dispense/dispense-common.ts

```diff
@@ -3,6 +3,15 @@ import {getExtensionForUrl, getNumericValueAsString} from "../../common"
 import {OrganisationTypeCode} from "../../common/organizationTypeCode"
 import {parseNumberOfRepeatsAllowed} from "../prescription"
 
+export function createAgentOrganisationFromOrganisation(
```

Comment:
looks similar to function below - can we remove?

Reply:
gone